src/Kiota.Builder/Writers/Swift/SwiftConventionService.cs

```diff
@@ -101,7 +101,7 @@ public override string TranslateType(CodeType type)
             "void" or "uint8" or "int8" or "int32" or "int64" or "float32" or "float64" or "string" => type.Name.ToFirstCharacterUpperCase(),
             "binary" => "[UInt8]",
             "DateTimeOffset" => "Date", // TODO
```

Comment:
What is this todo for?

Reply:
outside of the scope of this PR, resolving